### ISSUE TITLE
getHistoricalTrades - Too many parameters; expected '3' and received '4'.

### ISSUE THREAD

**Issue by corentindenoeud** [tiagosiebler/binance#245]
When i try : 

`          client.getHistoricalTrades({symbol: 'ETHUSDC'})`

I got this error message : 

```
  body: {
    code: -1101,
    msg: "Too many parameters; expected '3' and received '4'."
  }
```

**Comment by tiagosiebler** [comment]
Which client is this? Main or usdm?

**Comment by corentindenoeud** [comment]
On Main

**Comment by tiagosiebler** [comment]
Apologies for the delay. Looks like I unintentionally marked this endpoint as private, so we were sending a signature etc in that request. Binance apparently doesnt like that. 

Fix will be available shortly in v2.1.4, as soon as #250 is merged and CI finishes. Thanks for reporting this.
